packages/boiler-utils/src/remove-end-slashes.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Normalize for leading and trailing slashes
+ * @param {String} fp
+ *
+ * @return {String}
+ */
+export default function stripSlashes(fp) {
+  if (fp.length <= 1) return fp;
+
+  const len = fp.length;
```

Comment:
I know regular expressions are generally gross, but one could be handy here:

`return fp.trim().replace(/^\/|\/$/g, '');`